inttest/kubectl/kubectl_test.go

```diff
@@ -0,0 +1,130 @@
+package kubectl
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/k0sproject/k0s/inttest/common"
+	"github.com/stretchr/testify/suite"
+)
+
+type KubectlSuite struct {
+	common.FootlooseSuite
+}
+
+const pluginContent = `#!/bin/bash
+
+echo "foo-plugin"
+`
+
+func (s *KubectlSuite) TestEmbeddedKubectl() {
+	s.Require().NoError(s.InitController(0, "--enable-k0s-cloud-provider", "--k0s-cloud-provider-update-frequency=5s"))
```

Comment:
Do you really need `k0s-cloud-provider`?

Reply:
not at all, some copy-paste left-overs from check-basic